fili-core/src/main/java/com/yahoo/bard/webservice/util/IntervalUtils.java

```diff
@@ -88,7 +88,7 @@
      * @return A set of intervals describing the time common to both sets
      */
     public static Set<Interval> getOverlappingSubintervals(Set<Interval> left, Set<Interval> right) {
```

Comment:
@michael-mclawhorn Is this method redundant?  Even without it calling `getOverlappingSubintervals` with two sets will call the Collection one.